pyproject.toml

```diff
@@ -40,11 +40,11 @@ pgvector = { version = "^0.2.3", optional = true }
 psycopg = { version = "^3.1.12", optional = true }
 psycopg-binary = { version = "^3.1.12", optional = true }
 psycopg2-binary = { version = "^2.9.9", optional = true }
-torch = { version = "2.1.0", optional = true }
 huggingface-hub = { version = "0.17.3", optional = true }
 transformers = { version = "4.34.1", optional = true }
 pre-commit = {version = "^3.5.0", optional = true }
 pg8000 = {version = "^1.30.3", optional = true}
+torch = {version = ">=2.0.0, !=2.0.1, !=2.1.0", optional = true}
```

Comment:
Is there a reason we're downgrading the version here?

Reply:
one of the tests was failing because of it

https://stackoverflow.com/questions/76327419/valueerror-libcublas-so-0-9-not-found-in-the-system-path
![image](https://github.com/cpacker/MemGPT/assets/5475622/7f78a47d-ba57-4c6f-810a-90fdc13afa9d)

Reply:
downgrade is the stackoverflow solution